tests/foreman/cli/test_contentviews.py

```diff
@@ -86,20 +93,13 @@ def create_rhel_content(self):
         })
 
         RepositorySet.enable({
-            'name': (
-                'Red Hat Enterprise Virtualization Agents '
-                'for RHEL 6 Workstation (RPMs)'
-            ),
+            'name': REPOSET['rhva6'],
             'organization-id': ContentViewTestCase.rhel_content_org['id'],
-            'product': 'Red Hat Enterprise Linux Workstation',
-            'releasever': '6Workstation',
+            'product': PRDS['rhel'],
+            'releasever': '6Server',
```

Comment:
Not a blocker for this PR since I need it merged quickly, but I think we should use constants for `releasever` and `basearch` just like you did for `product`.